lib/services/tendermintCore/TendermintCoreOptions.js

```diff
@@ -0,0 +1,67 @@
+const DockerServiceOptions = require('../../docker/DockerServiceOptions');
+
+class TendermintCoreOptions extends DockerServiceOptions {
+  static setDefaultCustomOptions(options) {
+    TendermintCoreOptions.defaultCustomOptions = options;
+  }
+
+  mergeWithDefaultOptions(...customOptions) {
+    this.volumeId = Math.random().toString(36).substring(2, 15);
+
+    const defaultPorts = {
+      port: this.getRandomPort(26656, 26664),
+    };
+
+    const defaultServiceOptions = {
+      port: defaultPorts.port,
+    };
+
+    const abciUrl = customOptions[0].abciUrl || 'tcp://127.0.0.1:26658';
+
+    const cmd = ['node', '--rpc.laddr=tcp://0.0.0.0:26657', `--proxy_app=${abciUrl}`];
+
+    const defaultContainerOptions = {
+      image: 'tendermint/tendermint:latest',
+      network: {
+        name: 'dash_test_network',
+        driver: 'bridge',
+      },
+      ports: [
+        `${defaultServiceOptions.port}:26657`,
```

Comment:
I think for new services we can get rid of this stupid random port logic. We can specify an internal port only and then docker will choose an external one for us. Then we can just get it using inspect.